src/app.tsx

```diff
@@ -177,7 +177,9 @@ export const App = () => {
 			</Route>
 
 			<Route element={<SettingsLayout />} path="organization-settings">
+				<Route element={<OrganizationSettings />} index />
 				<Route element={<NewOrganization />} path="new" />
+				<Route element={<NewOrganization />} path="new-organization" />
```

Comment:
Everything is super good, the only comment that I have is that we have 2 routes now for NewOrganization:
<Route element={<NewOrganization />} path="new" />
<Route element={<NewOrganization />} path="new-organization" />

Reply:
removed last